### DISCUSSION TITLE
multiple PostgreSQL instances

### DISCUSSION THREAD

**Discussion by armudo** [heterodb/pg-strom#524] [Questions & Reports]
Can multiple PostgreSQL instances be used with a single GPU board?

**Comment by kaigai** [comment]
From the standpoint of OS resource management, it is equivalent to run parallel queries on a single PostgreSQL instance.
Because background workers are individual processes, with individual resources including GPU device context, thus, we have
no reason why to restrict to use a single GPU board by the multiple PostgreSQL instances.

On the other hands, we usually don't test PG-Strom on the configuration as you mentioned, and we don't doubt it makes
difficult to predicate resource consumption.

**Comment by armudo** [comment]
Thank you for answering my question!
